src/Aks/Aks/Commands/CreateOrUpdateKubeBase.cs

```diff
@@ -227,6 +226,21 @@ protected AcsServicePrincipal EnsureServicePrincipal(string spId = null, string
                     clientSecret = RandomBase64String(16);
                 }
 
+                bool supressWarningOrError = false;
+
+                try
+                {
+                    supressWarningOrError = bool.Parse(Environment.GetEnvironmentVariable(SUPPRESS_ERROR_OR_WARNING_MESSAGE_ENV_VARIABLE_NAME));
+                }
+                catch (Exception)
+                {
+                    //no action
+                }
+                if (!supressWarningOrError)
+                {
+                    WriteVerbose(Constants.MSGraphMigrationMessage);
```

Comment:
If the log level is verbose I think it's meaningless to support suppressing - so I think it should at least be informational (or warning) - let's discuss this later